src/CardLink.js

```diff
@@ -0,0 +1,34 @@
+import classNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+import elementType from 'prop-types-extra/lib/elementType';
+
+import { createBootstrapComponent } from './ThemeProvider';
+
+class CardLink extends React.Component {
+  static propTypes = {
+    /**
+     * @default 'card-link'
+     */
+    bsPrefix: PropTypes.string,
+
+    componentClass: elementType
+  };
+
+  static defaultProps = {
+    componentClass: 'a'
+  };
+
+  render() {
+    const {
+      bsPrefix,
+      className,
+      componentClass: Component,
+      ...props
+    } = this.props;
+
+    return <Component className={classNames(bsPrefix, className)} {...props} />;
+  }
+}
+
+export default createBootstrapComponent(CardLink, 'card-link');
```

Comment:
for these simple components I think there is a util for creating them dynamically, createWithBsPrefix i think? in /utils, might be easier for the logic-less ones

Reply:
I was thinking the same thing! I should have looked into the code more, thanks for the tip!